d.ts/anydb-sql.d.ts

```diff
@@ -251,6 +251,9 @@ declare module "anydb-sql-2" {
       open(): void;
       close(): void;
       getPool(): AnyDBPool;
+      setPool(pool: AnydbSql): void;
+      testMode(val: boolean): void;
```

Comment:
Rename `val` to `enable` at least? Or wrap the function into `startTest() { db.testMode(true) }` / `endTest() { db.testMode(false) }`.